api-report/fluid-static.api.md

```diff
@@ -18,6 +18,16 @@ import { IFluidDataStoreFactory } from '@fluidframework/runtime-definitions';
 import { IFluidLoadable } from '@fluidframework/core-interfaces';
 import { TypedEventEmitter } from '@fluidframework/common-utils';
 
+// @public
+export namespace ConnectionState {
```

Comment:
I'd really love to see us change names here:
- "Connecting" state should be actually call "Connected".
- "Connected" should be renamed to something else that clearly articulates actual state - "up-to-date & syncing changes".

Also, enums are expensive in JS, and I've heard advice to use union types only (as on line 29).

Reply:
I'd also suggest to re-export - it's cheaper that way.

Reply:
I copied these values from container-definitions in order to make `IContianer` and `IFluidContainer` consistent, however we can actually just import `ConnectionState` from container-definitions directly. As per changing the names, I believe that would be a breaking change, so we should probably do that in a separate PR.

Reply:
It would be great to have discussion on naming here. If we agree to change names, then I'd rather see you use new names here (doing remapping), not to create more back compat debt.

Reply:
> * "Connected" should be renamed to something else that clearly articulates actual state - "up-to-date & syncing changes".

@vladsud, I think `Synced` could be an adequate name. My only concern is that it does not emphasize that the container is actively syncing changes, however, I think most developers should be able to infer this.

Reply:
@vladsud, on a related note, do you know why we create  `ConnectionState` in [container.ts](https://github.com/microsoft/FluidFramework/blob/a8879b3e24c5188c5d89556b9abcb0bc6689480c/packages/loader/container-loader/src/container.ts#L143) instead of just importing the one defined in [container-definitions](https://github.com/microsoft/FluidFramework/blob/a8879b3e24c5188c5d89556b9abcb0bc6689480c/common/lib/container-definitions/src/loader.ts#L149)?

Reply:
I do not know, I'd guess someone added it there, likely as result of adding method to IContainer. Likely before that work there was no need to have it in definitions package.

Reply:
@markfields for naming discussion. I'm fine with "Synced", but let's run it by wider audience.

Reply:
For more naming discussion: https://github.com/microsoft/FluidFramework/issues/9677